script.js

```diff
@@ -0,0 +1,49 @@
+const totalSizeLimit = 15;
```

Comment:
What is 15? 15 what? MB? KB? GB?
Consider switching to bytes and converting to MB only when you update the DOM